doc/service_config.md

```diff
@@ -8,134 +8,30 @@ parameters to be automatically used by all clients of their service.
-      // set here. However, note that setting the value in the client API will
-      // also affect transient errors encountered during name resolution,
-      // which cannot be caught by the value here, since the service config
-      // is obtained by the gRPC client via name resolution.
-      'waitForReady': bool,
-
-      // Optional. The default timeout in seconds for RPCs sent to this method.
-      // This can be overridden in code. If no reply is received in the
-      // specified amount of time, the request is aborted and a
-      // deadline-exceeded error status is returned to the caller.
-      //
-      // The actual deadline used will be the minimum of the value specified
-      // here and the value set by the application via the gRPC client API.
-      // If either one is not set, then the other will be used.
-      // If neither is set, then the request has no deadline.
-      //
-      // The format of the value is that of the 'Duration' type defined here:
-      // https://developers.google.com/protocol-buffers/docs/proto3#json
-      'timeout': string,
-
-      // Optional. The maximum allowed payload size for an individual request
-      // or object in a stream (client->server) in bytes. The size which is
-      // measured is the serialized, uncompressed payload in bytes. This
-      // applies both to streaming and non-streaming requests.
-      //
-      // The actual value used is the minimum of the value specified here and
-      // the value set by the application via the gRPC client API.
-      // If either one is not set, then the other will be used.
-      // If neither is set, then the built-in default is used.
-      //
-      // If a client attempts to send an object larger than this value, it
-      // will not be sent and the client will see an error.
-      // Note that 0 is a valid value, meaning that the request message must
-      // be empty.
-      'maxRequestMessageBytes': number,
-
-      // Optional. The maximum allowed payload size for an individual response
-      // or object in a stream (server->client) in bytes. The size which is
-      // measured is the serialized, uncompressed payload in bytes. This
-      // applies both to streaming and non-streaming requests.
-      //
-      // The actual value used is the minimum of the value specified here and
-      // the value set by the application via the gRPC client API.
-      // If either one is not set, then the other will be used.
-      // If neither is set, then the built-in default is used.
-      //
-      // If a server attempts to send an object larger than this value, it
-      // will not be sent, and the client will see an error.
-      // Note that 0 is a valid value, meaning that the response message must
-      // be empty.
-      'maxResponseMessageBytes': number
-    }
-  ]
-}
-```
-
-Note that new per-method parameters may be added in the future as new
-functionality is introduced.
```

Comment:
Perhaps we should keep the JSON format example? A2-service-configs-in-dns.md also shows a JSON snippet. This will enable users to start quickly without understanding how to convert proto to json. May be move down in this doc?

Reply:
The format that was previously shown here was badly out of date, and I don't want to have to remember to update it every time we update the proto file, which is the authoritative source of truth.  But I have added a short example here in both protobuf and JSON form to illustrate how it should look.